src/System/Taffybar/Text/CPUMonitor.hs

```diff
@@ -18,12 +22,18 @@ textCpuMonitorNew fmt period = do
   where
     callback = do
       (userLoad, systemLoad, totalLoad) <- cpuLoad
+      cpuTemp <- getCPUTemp["cpu0"]
       let [userLoad', systemLoad', totalLoad'] = map (formatPercent.(*100)) [userLoad, systemLoad, totalLoad]
+      let [cpuTemp'] = map (showTemp) [cpuTemp]
       let template = ST.newSTMP fmt
       let template' = ST.setManyAttrib [ ("user", userLoad'),
                                          ("system", systemLoad'),
-                                         ("total", totalLoad') ] template
+                                         ("total", totalLoad'),
+					 ("temp",cpuTemp')] template
```

Comment:
documentation string needs to be updated in the usage. otherwise: nice job, i've been looking for something like this.